src/Api/AdminConsole/Public/Models/Request/AssociationWithPermissionsRequestModel.cs

```diff
@@ -1,17 +1,27 @@
-﻿using Bit.Core.Models.Data;
+﻿using Bit.Core.Exceptions;
+using Bit.Core.Models.Data;
 
 namespace Bit.Api.AdminConsole.Public.Models.Request;
 
 public class AssociationWithPermissionsRequestModel : AssociationWithPermissionsBaseModel
 {
-    public CollectionAccessSelection ToCollectionAccessSelection()
+    public CollectionAccessSelection ToCollectionAccessSelection(bool migratedToFlexibleCollections)
     {
-        return new CollectionAccessSelection
+        var collectionAccessSelection =  new CollectionAccessSelection
         {
             Id = Id.Value,
             ReadOnly = ReadOnly.Value,
-            HidePasswords = HidePasswords.GetValueOrDefault(),
-            Manage = Manage.GetValueOrDefault()
+            HidePasswords = HidePasswords.GetValueOrDefault()
         };
+
+        // Throws if the org has not migrated to use FC but has passed in a Manage value in the request
+        if (!migratedToFlexibleCollections && Manage.HasValue)
+        {
+            throw new BadRequestException(
+                "Your organization has not migrated to use Flexible Collections and cannot make use of the Manage property.");
+        }
+
+        collectionAccessSelection.Manage = Manage.GetValueOrDefault();
```

Comment:
You could still do this where you had it in the `CollectionAccessSelection` initializer. That way all the property assignments are together, and you still throw if it's not allowed.

Reply:
Fair!